src/libcore/cmp.rs

```diff
@@ -699,6 +699,39 @@ mod impls {
 
     ord_impl! { char usize u8 u16 u32 u64 isize i8 i16 i32 i64 }
 
+    // Note: This macro is a temporary hack that can be remove once we are building with a compiler
+    // that supports `!`
+    macro_rules! argh {
```

Comment:
This can be a bit more formal, e.g. "Remove this in the next snapshot." and `not_stage0` instead of `args`.
cc @alexcrichton Does "SNAP" work anymore?